package/yast2-schema.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Tue Feb  2 07:39:54 UTC 2021 - Knut Anderssen <kanderssen@suse.com>
+
+- Add networking interface section 'lladdr' element (bsc#1179876)
```

Comment:
It looks confusing to me. What about smoething like *Add the 'lladdr' element to the networking interface section*?